From 351c7dc9fa6fea65938f0de4e1cc445ad1771ec3 Mon Sep 17 00:00:00 2001
From: Mathieu Bridon <bochecha@daitauha.fr>
Date: Wed, 24 Jan 2018 18:37:15 +0100
Subject: [PATCH] Do not use hyphen for the reverse-DNS names

The convention for DBus and Flatpak is to not use hyphens in the last
segment of a name.

In fact, this is even illegal in object paths.

This commit moves everything from org.gnome.d-feet to org.gnome.dfeet
---
 configure.ac                                          |   2 +-
 data/Makefile.am                                      |   6 +++---
 .../{org.gnome.d-feet.svg => org.gnome.dfeet.svg}     |   0
 data/icons/Makefile.am                                |  18 +++++++++---------
 .../{org.gnome.d-feet.png => org.gnome.dfeet.png}     | Bin
 .../{org.gnome.d-feet.png => org.gnome.dfeet.png}     | Bin
 .../{org.gnome.d-feet.png => org.gnome.dfeet.png}     | Bin
 .../{org.gnome.d-feet.png => org.gnome.dfeet.png}     | Bin
 .../{org.gnome.d-feet.png => org.gnome.dfeet.png}     | Bin
 .../{org.gnome.d-feet.svg => org.gnome.dfeet.svg}     |   0
 ....appdata.xml.in => org.gnome.dfeet.appdata.xml.in} |   2 +-
 ...et.desktop.in.in => org.gnome.dfeet.desktop.in.in} |   2 +-
 ...d-feet.gschema.xml => org.gnome.dfeet.gschema.xml} |   2 +-
 data/ui/bus.ui                                        |   2 +-
 po/POTFILES.in                                        |   6 +++---
 po/POTFILES.skip                                      |   2 +-
 src/dfeet/application.py                              |   2 +-
 src/tests/tests.py                                    |   6 +++---
 19 files changed, 26 insertions(+), 26 deletions(-)
 rename data/icons/HighContrast/scalable/apps/{org.gnome.d-feet.svg => org.gnome.dfeet.svg} (100%)
 rename data/icons/hicolor/16x16/apps/{org.gnome.d-feet.png => org.gnome.dfeet.png} (100%)
 rename data/icons/hicolor/24x24/apps/{org.gnome.d-feet.png => org.gnome.dfeet.png} (100%)
 rename data/icons/hicolor/256x256/apps/{org.gnome.d-feet.png => org.gnome.dfeet.png} (100%)
 rename data/icons/hicolor/32x32/apps/{org.gnome.d-feet.png => org.gnome.dfeet.png} (100%)
 rename data/icons/hicolor/48x48/apps/{org.gnome.d-feet.png => org.gnome.dfeet.png} (100%)
 rename data/icons/hicolor/scalable/apps/{org.gnome.d-feet.svg => org.gnome.dfeet.svg} (100%)
 rename data/{org.gnome.d-feet.appdata.xml.in => org.gnome.dfeet.appdata.xml.in} (96%)
 rename data/{org.gnome.d-feet.desktop.in.in => org.gnome.dfeet.desktop.in.in} (91%)
 rename data/{org.gnome.d-feet.gschema.xml => org.gnome.dfeet.gschema.xml} (51%)
 rename org.gnome.d-feet.json => org.gnome.dfeet.json (98%)

diff --git a/configure.ac b/configure.ac
index bdeee4c..48102ea 100644
--- a/configure.ac
+++ b/configure.ac
@@ -48,7 +48,7 @@ AC_CONFIG_FILES([
 	src/dfeet/Makefile
 	src/tests/Makefile
 	data/Makefile
-	data/org.gnome.d-feet.desktop.in
+	data/org.gnome.dfeet.desktop.in
 	data/icons/Makefile
 	data/ui/Makefile
 	help/Makefile
diff --git a/data/Makefile.am b/data/Makefile.am
index 4c7e363..c50d1b6 100644
--- a/data/Makefile.am
+++ b/data/Makefile.am
@@ -1,12 +1,12 @@
 SUBDIRS = icons ui
 
-gsettings_SCHEMAS = org.gnome.d-feet.gschema.xml
+gsettings_SCHEMAS = org.gnome.dfeet.gschema.xml
 @GSETTINGS_RULES@
 
 @INTLTOOL_DESKTOP_RULE@
 
 desktopdir = $(datadir)/applications
-desktop_in_files = org.gnome.d-feet.desktop.in
+desktop_in_files = org.gnome.dfeet.desktop.in
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 
 UPDATE_DESKTOP = if test -z "$(DESTDIR)"; then update-desktop-database $(datadir)/applications || :; fi
@@ -14,7 +14,7 @@ UPDATE_DESKTOP = if test -z "$(DESTDIR)"; then update-desktop-database $(datadir
 @INTLTOOL_XML_RULE@
 appdatadir = $(datadir)/metainfo
 appdata_DATA = $(appdata_in_files:.xml.in=.xml)
-appdata_in_files = org.gnome.d-feet.appdata.xml.in
+appdata_in_files = org.gnome.dfeet.appdata.xml.in
 
 
 install-data-hook:
diff --git a/data/icons/HighContrast/scalable/apps/org.gnome.d-feet.svg b/data/icons/HighContrast/scalable/apps/org.gnome.dfeet.svg
similarity index 100%
rename from data/icons/HighContrast/scalable/apps/org.gnome.d-feet.svg
rename to data/icons/HighContrast/scalable/apps/org.gnome.dfeet.svg
diff --git a/data/icons/Makefile.am b/data/icons/Makefile.am
index 4e7c21d..5439998 100644
--- a/data/icons/Makefile.am
+++ b/data/icons/Makefile.am
@@ -1,6 +1,6 @@
 icon16dir = $(datadir)/icons/hicolor/16x16/apps
 icon16_DATA = \
-	hicolor/16x16/apps/org.gnome.d-feet.png \
+	hicolor/16x16/apps/org.gnome.dfeet.png \
 	hicolor/16x16/apps/dfeet-method-category.png \
 	hicolor/16x16/apps/dfeet-method.png \
 	hicolor/16x16/apps/dfeet-object.png \
@@ -11,28 +11,28 @@ icon16_DATA = \
 	hicolor/16x16/apps/dfeet-icon-default-service.png
 
 icon24dir = $(datadir)/icons/hicolor/24x24/apps
-icon24_DATA = hicolor/24x24/apps/org.gnome.d-feet.png
+icon24_DATA = hicolor/24x24/apps/org.gnome.dfeet.png
 
 icon32dir = $(datadir)/icons/hicolor/32x32/apps
-icon32_DATA = hicolor/32x32/apps/org.gnome.d-feet.png
+icon32_DATA = hicolor/32x32/apps/org.gnome.dfeet.png
 
 icon48dir = $(datadir)/icons/hicolor/48x48/apps
-icon48_DATA = hicolor/48x48/apps/org.gnome.d-feet.png
+icon48_DATA = hicolor/48x48/apps/org.gnome.dfeet.png
 
 #icon64dir = $(datadir)/icons/hicolor/64x64/apps
-#icon64_DATA = hicolor/64x64/apps/org.gnome.d-feet.png
+#icon64_DATA = hicolor/64x64/apps/org.gnome.dfeet.png
 
 #icon128dir = $(datadir)/icons/hicolor/128x128/apps
-#icon128_DATA = hicolor/128x128/apps/org.gnome.d-feet.png
+#icon128_DATA = hicolor/128x128/apps/org.gnome.dfeet.png
 
 icon256dir = $(datadir)/icons/hicolor/256x256/apps
-icon256_DATA = hicolor/256x256/apps/org.gnome.d-feet.png
+icon256_DATA = hicolor/256x256/apps/org.gnome.dfeet.png
 
 scalabledir = $(datadir)/icons/hicolor/scalable/apps
-scalable_DATA = hicolor/scalable/apps/org.gnome.d-feet.svg
+scalable_DATA = hicolor/scalable/apps/org.gnome.dfeet.svg
 
 scalablehcdir = $(datadir)/icons/HighContrast/scalable/apps
-scalablehc_DATA = HighContrast/scalable/apps/org.gnome.d-feet.svg
+scalablehc_DATA = HighContrast/scalable/apps/org.gnome.dfeet.svg
 
 
 gtk_update_icon_cache = gtk-update-icon-cache -f -t $(datadir)/icons/hicolor
diff --git a/data/icons/hicolor/16x16/apps/org.gnome.d-feet.png b/data/icons/hicolor/16x16/apps/org.gnome.dfeet.png
similarity index 100%
rename from data/icons/hicolor/16x16/apps/org.gnome.d-feet.png
rename to data/icons/hicolor/16x16/apps/org.gnome.dfeet.png
diff --git a/data/icons/hicolor/24x24/apps/org.gnome.d-feet.png b/data/icons/hicolor/24x24/apps/org.gnome.dfeet.png
similarity index 100%
rename from data/icons/hicolor/24x24/apps/org.gnome.d-feet.png
rename to data/icons/hicolor/24x24/apps/org.gnome.dfeet.png
diff --git a/data/icons/hicolor/256x256/apps/org.gnome.d-feet.png b/data/icons/hicolor/256x256/apps/org.gnome.dfeet.png
similarity index 100%
rename from data/icons/hicolor/256x256/apps/org.gnome.d-feet.png
rename to data/icons/hicolor/256x256/apps/org.gnome.dfeet.png
diff --git a/data/icons/hicolor/32x32/apps/org.gnome.d-feet.png b/data/icons/hicolor/32x32/apps/org.gnome.dfeet.png
similarity index 100%
rename from data/icons/hicolor/32x32/apps/org.gnome.d-feet.png
rename to data/icons/hicolor/32x32/apps/org.gnome.dfeet.png
diff --git a/data/icons/hicolor/48x48/apps/org.gnome.d-feet.png b/data/icons/hicolor/48x48/apps/org.gnome.dfeet.png
similarity index 100%
rename from data/icons/hicolor/48x48/apps/org.gnome.d-feet.png
rename to data/icons/hicolor/48x48/apps/org.gnome.dfeet.png
diff --git a/data/icons/hicolor/scalable/apps/org.gnome.d-feet.svg b/data/icons/hicolor/scalable/apps/org.gnome.dfeet.svg
similarity index 100%
rename from data/icons/hicolor/scalable/apps/org.gnome.d-feet.svg
rename to data/icons/hicolor/scalable/apps/org.gnome.dfeet.svg
diff --git a/data/org.gnome.d-feet.appdata.xml.in b/data/org.gnome.dfeet.appdata.xml.in
similarity index 96%
rename from data/org.gnome.d-feet.appdata.xml.in
rename to data/org.gnome.dfeet.appdata.xml.in
index 87c4bb6..c4065c1 100644
--- a/data/org.gnome.d-feet.appdata.xml.in
+++ b/data/org.gnome.dfeet.appdata.xml.in
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!-- Copyright 2013 Thomas Bechtold <thomasbechtold@jpberlin.de> -->
 <component type="desktop">
-  <id>org.gnome.d-feet.desktop</id>
+  <id>org.gnome.dfeet.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
   <project_license>GPL-2.0+</project_license>
   <_name>D-Feet</_name>
diff --git a/data/org.gnome.d-feet.desktop.in.in b/data/org.gnome.dfeet.desktop.in.in
similarity index 91%
rename from data/org.gnome.d-feet.desktop.in.in
rename to data/org.gnome.dfeet.desktop.in.in
index 6581f0c..134cf10 100644
--- a/data/org.gnome.d-feet.desktop.in.in
+++ b/data/org.gnome.dfeet.desktop.in.in
@@ -4,7 +4,7 @@ _GenericName=D-Bus Debugger
 _Comment=Debug D-Bus applications
 _Keywords=debug;d-bus;dbus;dfeet;
 Exec=@PACKAGE@
-Icon=org.gnome.d-feet
+Icon=org.gnome.dfeet
 Terminal=false
 Type=Application
 Categories=GNOME;GTK;Development;Debugger;
diff --git a/data/org.gnome.d-feet.gschema.xml b/data/org.gnome.dfeet.gschema.xml
similarity index 51%
rename from data/org.gnome.d-feet.gschema.xml
rename to data/org.gnome.dfeet.gschema.xml
index 297cd1f..d46272e 100644
--- a/data/org.gnome.d-feet.gschema.xml
+++ b/data/org.gnome.dfeet.gschema.xml
@@ -1,4 +1,4 @@
 <schemalist gettext-domain="d-feet">
-  <schema id="org.gnome.d-feet" path="/org/gnome/d-feet/">
+  <schema id="org.gnome.dfeet" path="/org/gnome/dfeet/">
   </schema>
 </schemalist>
diff --git a/data/ui/bus.ui b/data/ui/bus.ui
index 6708243..059e041 100644
--- a/data/ui/bus.ui
+++ b/data/ui/bus.ui
@@ -54,7 +54,7 @@
         <property name="visible">True</property>
         <property name="can_focus">False</property>
         <property name="pixel_size">128</property>
-        <property name="icon_name">org.gnome.d-feet</property>
+        <property name="icon_name">org.gnome.dfeet</property>
       </object>
       <packing>
         <property name="expand">True</property>
diff --git a/po/POTFILES.in b/po/POTFILES.in
index e73de53..4e31d24 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -1,8 +1,8 @@
 # List of source files containing translatable strings.
 # Please keep this file sorted alphabetically.
-data/org.gnome.d-feet.appdata.xml.in
-data/org.gnome.d-feet.desktop.in.in
-data/org.gnome.d-feet.gschema.xml
+data/org.gnome.dfeet.appdata.xml.in
+data/org.gnome.dfeet.desktop.in.in
+data/org.gnome.dfeet.gschema.xml
 [type: gettext/glade]data/ui/addconnectiondialog.ui
 [type: gettext/glade]data/ui/app-menu.ui
 [type: gettext/glade]data/ui/bus.ui
diff --git a/po/POTFILES.skip b/po/POTFILES.skip
index 45f9f4b..d0845fe 100644
--- a/po/POTFILES.skip
+++ b/po/POTFILES.skip
@@ -1 +1 @@
-data/org.gnome.d-feet.desktop.in
+data/org.gnome.dfeet.desktop.in
diff --git a/src/dfeet/application.py b/src/dfeet/application.py
index c8e02ee..9ddb66d 100644
--- a/src/dfeet/application.py
+++ b/src/dfeet/application.py
@@ -12,7 +12,7 @@ class DFeetApp(Gtk.Application):
         self.package = package
         self.version = version
         self.data_dir = data_dir
-        Gtk.Application.__init__(self, application_id="org.gnome.d-feet",
+        Gtk.Application.__init__(self, application_id="org.gnome.dfeet",
                                  flags=Gio.ApplicationFlags.FLAGS_NONE)
 
     # Note that the function in C activate() becomes do_activate() in Python
diff --git a/src/tests/tests.py b/src/tests/tests.py
index 4fb52d5..31a7292 100755
--- a/src/tests/tests.py
+++ b/src/tests/tests.py
@@ -16,7 +16,7 @@ import unittest
 
 XML = """
 <node>
-  <interface name='org.gnome.d-feet.TestInterface'>
+  <interface name='org.gnome.dfeet.TestInterface'>
     <method name='HelloWorld'>
       <arg type='s' name='greeting' direction='in'/>
       <arg type='s' name='response' direction='out'/>
@@ -38,8 +38,8 @@ DATA_DIR = os.path.abspath("../../data/")
 class IntrospectionHelperTest(unittest.TestCase):
     """tests for the introspection helper classes"""
     def setUp(self):
-        self.name = "org.gnome.d-feet"
-        self.object_path = "/org/gnome/d-feet"
+        self.name = "org.gnome.dfeet"
+        self.object_path = "/org/gnome/dfeet"
         self.node_info = Gio.DBusNodeInfo.new_for_xml(XML)
 
     def test_dbus_node_info(self):
-- 
2.14.3

